kip-0023.md

```diff
@@ -0,0 +1,438 @@
+---
+KIP: "0023"
+Title: WebAuthn signatures for Chainweb and Pact
+Author: Andy Tang @EnoF, Eileen Guo @eileenmguo
+Status: Draft
+Type: Standard
+Category: Interface
+Created: 2023-10-27
+---
+
+# Abstract
+
+We propose leveraging WebAuthn signatures as an alternative to ED25519
+signatures for creating passwordless accounts and approving transactions
+processed through Chainweb Node and Pact smart contracts.
+
+# Motivation
+
+The current user experience for creating an account and signing transactions can
+be challenging for users unfamiliar with blockchain technology:
+
+- The concept of public / private keypairs is not widely understood.
+- Wallets often generate a 12-24 word mnemonic phrase that users need to store
+  securely in addition to a traditional password.
+
+Moreover, the security of accounts generated through many existing wallets is
+centralized because the user's key pairs are generated and stored by the wallet
+provider. If the wallet provider becomes unavailable or is compromised, users
+may lose access to their assets.
```

Comment:
Are you serious, sir ?

Reply:
No this paragraph is inaccurate and is being rewritten.